pwa-devdocs/src/peregrine/index.md

```diff
@@ -2,9 +2,9 @@
 title: Peregrine
 ---
 
-The Peregrine project is a collection of UI components for Magento PWA projects.
+The Peregrine project is a collection of UI Components for Magento PWA projects.
```

Comment:
The lowercase form of 'components' is the correct form in this context.
They are nothing more than components for UI.

Using 'UI Components' imply an official label for these components, which is not desired.